src/embezzler.ts

```diff
@@ -633,6 +635,30 @@ export const gregFights = (
   ];
 };
 
+const gregLikeSources = [
+  ...gregFights(
+    "Be Gregarious",
+    () => true, // we can always use extrovermectin
+    "beGregariousMonster",
+    "beGregariousFightsLeft",
+    () => get("beGregariousCharges") * 3 + get("beGregariousFightsLeft"),
+  ),
+  ...gregFights(
+    "Habitats Monster",
+    () => have($skill`Just the Facts`),
+    "_monsterHabitatsMonster",
+    "_monsterHabitatsFightsLeft",
+    () =>
+      have($skill`Just the Facts`)
+        ? (3 - get("_monsterHabitatsRecalled")) * 5 + get("_monsterHabitatsFightsLeft")
+        : 0,
+  ),
+];
```

Comment:
This looks like duplicated code, isn't it?

If you're going to make this its own source for counting, you should then plop this into the conditionalSources array, instead of us generating copies of the same fights there